src/main/webapp/college-info.html

```diff
@@ -1,21 +1,25 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>Black Student Insight</title>
+    <title>College Info Page</title>
     <meta charset="UTF-8">
     <link rel="stylesheet" href="style.css">
     <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
     <script src="script.js"></script>
   </head>
   <body>
     <header>
-      <h1>Black Student Insight</h1>
-      <input type="text" id="school-search">
-      <search-button onclick="loadSchoolInfo()">Search</search-button>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution">
+      <button id="search-button" onclick="loadSchoolInfo()">Search</button>
     </header>
     <main>
-      <h1 id="school-name"></h1>
+      <div class="topnav">
```

Comment:
Maybe use the `nav` tag? 

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav